modules/nextflow/src/main/groovy/nextflow/Session.groovy

```diff
@@ -126,6 +126,11 @@ class Session implements ISession {
      */
     Path workDir
 
+    /**
+     * The size of the filesystem to use for the work directory
+     */
+    int fsSize
```

Comment:
why does this need to be passed in through the pod spec as a label? 

does it makes sense to just pass this in as a variable when you provision the storage and then store that in the DB, this feels weird no?